content/cybersecurity/concepts/nmap/terms/nmap-Version-Detection/nmap-Version-Detection.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Can we change the filename from nmap-Version-Detection/nmap-Version-Detection.md to nmap-version-detection/nmap-version-detection.md?

Reply:
sure